### DISCUSSION TITLE
Remove icon and border color green when input is valid

### DISCUSSION THREAD

**Discussion by mathysth** [twbs/bootstrap#39970] [Q&A] [answered]
I'm trying to remove the icon from both valid and invalid state, and I also would like to remove the border color: green when an input is valid.
I'm using bootstrap 5.3.2 with angular
![image](https://github.com/twbs/bootstrap/assets/38434126/55499435-aac7-4b3d-80eb-f3f35ec2ca0b)

**Comment by patkon** [comment, marked as answer]
Have you tried adjusting the variables? in the _variables.scss file:
https://github.com/twbs/bootstrap/blob/72d3b6efc4b92e83a4abca6f5bc0cd7e6fd25931/scss/_variables.scss#L1086

`$form-feedback-icon-valid`
`$form-feedback-icon-invalid`
`$form-valid-border-color`

**Reply by julien-deramond** [reply]
As mentioned by @patkon, it's possible to override some Sass variables to do something like this:

```scss
@import "functions";

$form-feedback-icon-valid: "" !default;
$form-feedback-icon-invalid: "" !default;
$form-valid-border-color: var(--bs-border-color) !default;

@import "variables";

/* ... */
```
